app/controllers/categories_controller.rb

```diff
@@ -1,6 +1,7 @@
 class CategoriesController < ApplicationController
   def index
-    @categories = Category.all
+    @projects = Project.all.reject(&:retired)
```

Comment:
Again, I know that this is a change that I make in paperclip life... should I still merge this into master knowing that this line gets changed in my PR? The answer I imagine is yes?